frontend/web/components/OrganisationLimit.tsx

```diff
@@ -1,37 +1,43 @@
 import { FC } from 'react'
 import WarningMessage from './WarningMessage'
-import ErrorMessage from './ErrorMessage'
 import Utils from 'common/utils/utils'
 import { useGetSubscriptionMetadataQuery } from 'common/services/useSubscriptionMetadata'
 import Format from 'common/utils/format'
 import { useGetOrganisationUsageQuery } from 'common/services/useOrganisationUsage'
+import QuotaExceededMessage from './QuotaExceededMessage'
 
 type OrganisationLimitType = {
   id: string
+  organisationPlan: string
 }
 
-const OrganisationLimit: FC<OrganisationLimitType> = ({ id }) => {
-  const { data: totalApiCalls } = useGetOrganisationUsageQuery(
-    {
-      organisationId: id,
-    },
-    { skip: !id },
-  )
+const OrganisationLimit: FC<OrganisationLimitType> = ({
+  id,
+  organisationPlan,
+}) => {
+  let body = { organisationId: id }
+  if (Utils.getPlanName(organisationPlan) !== 'free') {
+    body = { ...body, ...{ billing_period: 'current_billing_period' } }
+  }
+
+  const { data: totalApiCalls } = useGetOrganisationUsageQuery(body, {
```

Comment:
Can’t this just go into the new component ?

Reply:
I'm not quite sure I understand your suggestion. Are you suggesting using this hook in the new QuotaExceededMessagecomponent?

Reply:
Well yes or just move the QuotaExceededMessagecomponent to this, it doesn't look like it's necessary

Reply:
Done